tests/layer_tests/ovc_python_api_tests/test_tf_output_order.py

```diff
@@ -67,13 +67,15 @@ class TestTFInputOutputOrder():
     def setup_method(self):
         Path(constants.out_path).mkdir(parents=True, exist_ok=True)
         self.tmp_dir = tempfile.TemporaryDirectory(dir=constants.out_path).name
-
-    @pytest.mark.parametrize("save_to_file, create_model_method, compare_model_method", [
+    test_data=[
         (False, create_net_list, check_outputs_by_order),
         (False, create_net_dict, check_outputs_by_names),
         pytest.param(True, create_net_list, check_outputs_by_order, marks=pytest.mark.xfail(reason='124436')),
         pytest.param(True, create_net_dict, check_outputs_by_names, marks=pytest.mark.xfail(reason='124436')),
-    ])
+    ]
+    test_ids = ["Test{}".format(id) for id in range(len(test_data))]
```

Comment:
why do we need `test_ids`? I think it was used for debugging and now can be removed. Right, @mahajanparth?